admin/admin-service/src/main/java/io/mosip/admin/bulkdataupload/service/PacketUploadService.java

```diff
@@ -0,0 +1,274 @@
+import io.mosip.kernel.core.util.HMACUtils2;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
+import org.springframework.stereotype.Component;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import javax.annotation.PostConstruct;
+import javax.transaction.Transactional;
+
+import java.io.File;
+import java.math.BigInteger;
+import java.net.URI;
+import java.time.LocalDateTime;
+import java.util.*;
+
+import static io.mosip.kernel.core.util.JsonUtils.javaObjectToJsonString;
+
+@Component
+public class PacketUploadService {
+
+    private static final Logger logger = LoggerFactory.getLogger(PacketUploadService.class);
+    private static final String PACKET_SYNC_STATUS_ID = "mosip.registration.sync";
+    private static final String PACKET_SYNC_VERSION = "1.0";
+    private static final String SPACE = " ";
+
+    @Value("${mosip.kernel.registrationcenterid.length:5}")
+    private int centerIdLength;
+    
+    @Value("${mosip.kernel.machineid.length:5}")
+    private int machineIdLength;
+    
+    @Autowired
+    private PacketRestClient restClient;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    @Qualifier("OnlinePacketCryptoServiceImpl")
+    private IPacketCryptoService onlineCrypto;
+    
+    @Autowired
+    private PacketReader packetReader;
+
+    @Autowired
+    private RestTemplate restTemplate;
+
+    @Value("${object.store.base.location:home}")
+    private String baseLocation;
+
+    @Value("${packet.manager.account.name:PACKET_MANAGER_ACCOUNT}")
+    private String account;
+
+    @Value("${mosip.admin.packetupload.packetsync.name}")
+    private String nameFieldNames;
+
+    @Value("${mosip.admin.packetupload.packetsync.email}")
+    private String emailFieldName;
+
+    @Value("${mosip.admin.packetupload.packetsync.phone}")
+    private String phoneFieldName;
+
+    @Value("${mosip.admin.packetupload.packetsync.url}")
+    private String packetSyncURL;
+
+    @Value("${mosip.kernel.packet-reciever-api-url}")
+    private String packetReceiverURL;
+
+    @Value("${mosip.mandatory-languages}")
+    private String mandatoryLanguages;
+
+    @Value("${mosip.optional-languages}")
+    private String optionalLanguages;
+
+    @Value("${MACHINE_GET_API}")
+    private String MACHINE_GET_API;
+
+    private String language;
+
+    @PostConstruct
+    public void init() throws InterruptedException {
+        File accountLoc = new File(baseLocation + File.separator + account);
```

Comment:
why not use  `new File(baseLocation, account)`

Reply:
updated it, sir.